rust/polars-ocaml/src/data_frame.rs

```diff
@@ -339,6 +350,37 @@ fn rust_data_frame_diagonal_concat(
         .to_ocaml(cr)
 }
 
+#[ocaml_interop_export]
+fn rust_data_frame_vstack(
+    cr: &mut &mut OCamlRuntime,
+    data_frame: OCamlRef<DynBox<PolarsDataFrame>>,
+    other: OCamlRef<DynBox<PolarsDataFrame>>,
+) -> OCaml<Result<DynBox<()>, String>> {
+    let Abstract(other) = other.to_rust(cr);
+    let other = match Rc::try_unwrap(other) {
+        Ok(other) => other.into_inner(),
+        Err(other) => other.borrow().clone(),
+    };
+
+    dyn_box_result!(cr, |data_frame| {
```

Comment:
dyn_box can take multiple arguments, so you can do `|data_frame, other| { ... }` and it'll just work; let's do that instead!

Reply:
done